torchtitan/experiments/deepseek_v3/model.py

```diff
@@ -454,6 +454,8 @@ class MoE(nn.Module):
     # 1. "torch_all_to_all"
     # 2. "symm_mem" (see `setup_symm_mem` below)
     shuffle_method = "torch_all_to_all"
+    # Group GEMM method, "torch" or "torchao"
+    group_mm = "torch"
```

Comment:
tiny nit but I like just having selectable options next to the item, like how we do it in the toml:
~~~
group_mm = "torch"  # ["torch", "torchao"]
~~~